frontend/src/services/api.js

```diff
@@ -1,100 +1,119 @@
-import axios from 'axios';
+import axios from "axios";
 
-const API_BASE_URL = 'http://localhost:8000/api'; 
+const API_BASE_URL = "http://localhost:8000/api";
```

Comment:
I thought laura merged code that moved this to an env variable? please check her latest pr